ui/raidboss/data/06-ew/raid/p3s.ts

```diff
@@ -320,6 +334,89 @@ const triggerSet: TriggerSet<Data> = {
         },
       },
     },
+    {
+      id: 'P3S Sunbird Tether',
+      type: 'Tether',
+      netRegex: NetRegexes.tether({ id: '0039' }),
+      promise: async (data) => {
+        const callData = await callOverlayHandler({
+          call: 'getCombatants',
```

Comment:
Is there some way to limit this call to not have it return every single combatant? Maybe by name, or npc id, or even if you already have the previous ids somewhere?

Reply:
I don't really know how to filter this. I see calls to getCombatants mostly not being filtered at all, sometimes by id but these have multiple so that'd be really complicated and I found https://github.com/quisquous/cactbot/blob/cf43f8fb13ae4f33cd1a271696f8df9dbd1c995d/ui/raidboss/data/05-shb/raid/e9s.ts#L842 doing it with a hard coded name array.

I could hardcode it but I have no idea what they are called in cn so this would break if locale is cn. Imo investing the additional memory copied to not depend on locale is worth it but your call obviously.